scripts/release_helper/go.py

```diff
@@ -3,22 +3,19 @@
 import os
 
 # assignee dict which will be assigned to handle issues
-_GO_OWNER = {'ArcturusZhang', 'Alancere'}
-
-# 'github assignee': 'token'
-_ASSIGNEE_TOKEN_GO = {'Alancere': os.getenv('AZURESDK_BOT_TOKEN')}
+_GO_OWNER = {'ArcturusZhang', 'Alancere', 'azure-sdk'}
```

Comment:
not all owners will handle issues. We still need two set:

- `LANGUAGE_ASSIGNEE`: person who handle issue
- `LANGUAGE_ADDITIONAL_OWNER`: person who is not assignee but may comment